inc/saiqueue.h

```diff
@@ -81,111 +79,108 @@ typedef enum _sai_queue_attr_t
     /* -- */
 
     SAI_QUEUE_ATTR_END,
-    
+
     /* Custom range base value */
     SAI_QUEUE_ATTR_CUSTOM_RANGE_START = 0x10000000,
 
     /* --*/
     SAI_QUEUE_ATTR_CUSTOM_RANGE_END
 
-
 } sai_queue_attr_t;
 
-
-
 /**
  * @brief Enum defining statistics for Queue.
  */
 
 typedef enum _sai_queue_stat_counter_t
```

Comment:
remove the _counter_ from _sai_queue_stat_counter_t

---
In reply to: [78670968](https://github.com/opencomputeproject/SAI/pull/208#discussion_r78670968) [](ancestors = 78670968)